zilliqa/src/consensus.rs

```diff
@@ -1525,7 +1526,14 @@ impl Consensus {
         self.verify_qc_signature(&block.qc, committee.clone())?;
         if let Some(agg) = &block.agg {
             // Check if the signers of the block's aggregate QC represent the supermajority
-            self.check_quorum_in_indices(&agg.signers, &committee)?;
+
+            let mut signers = Vec::new();
+            for (index, bit) in agg.signers.iter().enumerate() {
+                if *bit {
+                    signers.push(index as u16);
```

Comment:
You're taking one field called signers (different from the Vote field called signers) and turning it into  a variable called signers which contains different values.

Comments, please ! (and preferably let's name things differently - perhaps signer_index_vec<> and signer_bitvec<> ?)